src/modules/esl-media/core/esl-media-provider.ts

```diff
@@ -67,6 +67,10 @@ export abstract class BaseProvider {
     return this._ready;
   }
 
+  public set aspectRatio(ratio: number) {
+    this._el?.style.setProperty('aspect-ratio', ratio.toString());
+  }
+
```

Comment:
In case we have only one setter I'd recommend to place the exact set method nearby `setSize`

```
  public setAspectRatio(aspectRatio: number): void {
    this._el.style.setProperty('aspect-ratio', aspectRatio > 0 ? `${aspectRatio}`: null);
  }
```